### PR TITLE
add part of chr22

### PR DESCRIPTION
**Pull request by maxulysse**
for bbsplit
cf https://github.com/nf-core/modules/pull/771

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,6 +242,8 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
         - dbNSFP4.1a.21: Created from dbNSFP database. Chromosome 21 extracted from main file and posterior subsampling of first 100K lines.
         - SNP_GRCh38_hg38_wChr.bed: Common heterozygous SNPs, used to determine if samples match in the NGSCheckMate tool (chr21 only)
       - chromosomes.tar.gz: compressed directory containing the fasta genome file renamed to chr21 (needed for ControlFREEC)
+    - chr22: directory for reference files using chr22, for bbsplit
+      - sequence: directory containing fasta for chr22
```
Comment (JoseEspinosa) [inline README.md:246]:
```suggestion
      - chr22_23800000-23980000.fa: Fasta file containing a section of chr22
```

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Just a small suggestion

File: README.md [hunk 2]
```diff
@@ -242,6 +242,8 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
         - dbNSFP4.1a.21: Created from dbNSFP database. Chromosome 21 extracted from main file and posterior subsampling of first 100K lines.
         - SNP_GRCh38_hg38_wChr.bed: Common heterozygous SNPs, used to determine if samples match in the NGSCheckMate tool (chr21 only)
       - chromosomes.tar.gz: compressed directory containing the fasta genome file renamed to chr21 (needed for ControlFREEC)
+    - chr22: directory for reference files using chr22, for bbsplit
+      - chr22_23800000-23980000.fa: Fasta file containing a section of chr22
```
Comment (maxulysse) [inline README.md:246]:
```suggestion
      - squence/chr22_23800000-23980000.fa: Fasta file containing a section of chr22
```

**Review by JoseEspinosa — APPROVED** [review]
Thanks!

**Pull request merged**